12/umbraco-cms/tutorials/creating-custom-views-for-blocklist.md

```diff
@@ -161,7 +161,7 @@ Additionally, we need to update our stylesheet to use the color configuration. F
 @import url(https://fonts.googleapis.com/css?family=Montserrat|Source+Sans+Pro:400,700,300,600,600italic,400italic);
 
 body {
- font-family: 'Source Sans Pro', sans-seif;   
+ font-family: 'Source Sans Pro', sans-serif;   
```

Comment:
to keep the docs consistent, can you please make this change to v10 and v11 as well?